content/en/_index.md

```diff
@@ -1138,33 +1138,31 @@ end
 {{< /content-tab >}}
 
 {{< content-tab header="Terraform" >}}
-#### Don't use `null_resource`
+
+### Don't use `null_resource`
 
 Do not include CLI commands (such as gcloud or kubectl) inside
 of your sample via the [`null_resource`](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) resource.
 
 There are some products without Terraform support. Do not include
 those products in Terraform samples.
 
-#### No global variables
+### No global variables
 
 Don't include global Terraform variables (`var.VARIABLE_NAME`).
 
-{{% alert title="Explanation" %}}
```

Comment:
This was added at my request, because we generally exclude explanations from the style guide. I've been thinking that excluding explanations may be good for concise readability, but not for communicating "why", so maybe we need a better model to mostly hide but make available information like this.

What do you think about using something like a details element?


<details>
<summary>Explanation for this rule</summary>

Global variables become part of the API exposed via Terraform.
Therefore, global variables help enable users to use Terraform samples
not as snippets but as direct module references. 

</details>